### ISSUE TITLE
Cannot install glibc-devel.i686 in manylinux2010_x86_64

### ISSUE THREAD

**Issue by mephi42** [pypa/manylinux#561]
```
# yum install glibc-devel.i686
...
Error: Package: glibc-2.12-1.212.el6_10.3.i686 (updates)
           Requires: glibc-common = 2.12-1.212.el6_10.3
           Installed: glibc-common-2.12-1.212.1.el6.x86_64 (@/glibc-common-2.12-1.212.1.el6.x86_64)
```

This must be because centos-6-no-vsyscall builds only 64-bit glibc. While I understand that the primary purpose of manylinux images is to build python wheels for specific architectures, it is still nice to have the base system fully functioning.

What do you think about building and installing a 32-bit glibc like this https://github.com/mephi42/manylinux/commit/82a54a74173fd7c3a3d4ffa2044b257d9cdbcdfb?
